test_parse.py

```diff
@@ -205,6 +205,17 @@ def test_typed(self):
         r = parse.parse('hello {:w} {:w}', 'hello 12 people')
         self.assertEqual(r.fixed, ('12', 'people'))
 
+    def test_sign(self):
+        # sign is ignored
+        r = parse.parse('Pi = {:.7f}', 'Pi = 3.1415926')
+        self.assertEqual(r.fixed, (3.1415926,))
+        r = parse.parse('Pi = {:+.7f}', 'Pi = 3.1415926')
+        self.assertEqual(r.fixed, (3.1415926,))
+        r = parse.parse('Pi = {:-.7f}', 'Pi = 3.1415926')
+        self.assertEqual(r.fixed, (3.1415926,))
```

Comment:
I can't help thinking that the parsed value should be negative here.

Reply:
I'm not sure what you mean?  A `-` in the format string just means "use a minus for negative values, and nothing for positive values":
```
In [1]: "{:-.3f}".format(+1.23), "{:-.3f}".format(-1.23)
Out[1]: ('1.230', '-1.230')
```

Reply:
Oof. That's what I get for trying to do some speed-reviewing of PRs :( Indeed, that looks fine.